packages/vats/src/vat-board.js

```diff
@@ -10,7 +10,7 @@ const THE_BOARD = 'theboard';
 /**
  * @param {unknown} _vatPowers
  * @param {unknown} _vatParameters
- * @param {import('@agoric/vat-data').Baggage} baggage
+ * @param {import('@agoric/swingset-liveslots').Baggage} baggage
```

Comment:
I wondered whether exposing so many contracts etc. to `swingset-liveslots` was a problem. I looked at the depgraph from `scripts/graph.sh`. It shows `vat-data` is right next door, so it doesn't seem like a problem.